src/libcollections/btree/set.rs

```diff
@@ -96,6 +105,15 @@ pub struct IntoIter<T> {
     iter: ::btree_map::IntoIter<T, ()>,
 }
 
+#[stable(feature = "collection_debug", since = "1.16.0")]
+impl<T: fmt::Debug> fmt::Debug for IntoIter<T> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_tuple("BTreeSet::IntoIter")
```

Comment:
This should also just be `#[derive]`

Note that when I point things out like this it's typically useful to go over the PR looking for other cases as well. I'll try to exhaustively point it out this time but only fixing precisely what is pointed out isn't always the best strategy.